docs/en/ingest-management/faq.asciidoc

```diff
@@ -28,7 +27,9 @@ command to start it.
 When started successfully, {metricbeat} logs are stored in
 `data/logs/metricbeat` under the folder where {agent} was started. If that log
 path does not exist, {agent} was unable to start {metricbeat}, which is a
-higher level problem to triage.
+higher level problem to triage.  Usually you can see these logs in the Fleet
+UI, unless there are problems severe enough that the processes cannot send 
+data to Elasticsearch.
```

Comment:
```suggestion
higher level problem to triage. Usually you can see these logs in the {fleet}
UI, unless there are problems severe enough that the processes cannot send 
data to {es}.
```

Reply:
"the processes" seems a little vague. Can we just say "the agent cannot send"? I think maybe you are talking about the underlying Beats shippers, but I wonder if we can generalize here. Or maybe we can say something like "severe enough that the agent or its related processes cannot send..."

Reply:
I like your latter note, I'll do that.

Reply:
done